app/components/ui/Navigation.tsx

```diff
@@ -41,29 +41,40 @@ export const Navigation = ({
 
         <ul className={styles.navRight}>
           {menus?.map((menu) => (
-            <div className={styles.menuContainer} key={menu.id.toString()}>
+            <div
+              className={styles.navigationMenuContainer}
+              key={menu.id.toString()}
+            >
               <button
                 type="button"
                 aria-haspopup="menu"
-                aria-expanded={dropdown ? "true" : "false"}
+                aria-expanded={dropdownIsOpen ? "true" : "false"}
                 onClick={() => setDropdown((prev) => !prev)}
+                className={styles.navigationMenuHeader}
               >
                 {menu.title}
+                <span className={`fas fa-chevron-down ${styles.chevron}`} />
               </button>
 
               <ul
-                className={`${styles.dropdown} ${
-                  dropdown ? styles.showDropdown : ""
+                className={`${styles.dropdown} ${styles.navigationMenuList} ${
```

Comment:
I'm curious why `.dropdown`'s only css is `display: none`? Is there a more descriptive classname we can use?

Reply:
Copy/paste. I'll iterate on this, nice 👀